src/components/shared/TooltipHeaderWithDuration.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  duration: number,
+  title: string,
+  className?: string,
+};
+
+export default class TooltipHeaderWithDuration extends PureComponent {
```

Comment:
true story: I started by MarkerTooltipContents, then MarkerTooltipHeader, and then ended up with this name.

But I think you're right, we should share this only for marker tooltips -- if we share it at all.